scripts/pin_the_slow_way.py

```diff
@@ -62,6 +62,7 @@
           'libssh2': 'libssh2 1.8.*',  # 1.8.0
           'libsvm': 'libsvm 3.21|3.21.*',  # NA
           'libtiff': 'libtiff >=4.0.3,<4.0.8',  # 4.0.6
+          'libwebp': 'libwebp 0.5.*',  # 0.5.2 https://abi-laboratory.pro/tracker/timeline/libwebp/
```

Comment:
`0.5.2` -> `0.5.0` (defaults has 0.5.0 and the commented versions show which version defaults channel is using)